lib/suse/connect/cli.rb

```diff
@@ -117,14 +118,17 @@ def extract_options # rubocop:disable MethodLength
           @options[:product] = Remote::Product.new(identifier: identifier, version: version, arch: arch)
         end
 
-        @opts.on('-r', '--regcode [REGCODE]', 'Subscription registration code for the product to',
+        @opts.on('-r', '--regcode [REGCODE]',
+                 'Subscription registration code for the product to',
                  'be registered.',
                  'Relates that product to the specified subscription,',
                  'and enables software repositories for that product.') do |opt|
           @options[:token] = opt
         end
 
-        @opts.on('-d', '--de-register', 'De-registers a system and removes all services',
+        @opts.on('-d', '--de-register',
+                 'De-registers a system or a single product (if',
```

Comment:
'De-registers the system and base product, or in conjunction with --product, a single addon/extension product, and removes...'

note change to 'the system' not a system in any case.